### PR TITLE
Added logout feature

### PR DESCRIPTION
**Pull request by piyush0**
Signed-off-by: Piyush Ajmani <piyushajmani5@gmail.com>

Fixes Issue #18

### CODE REVIEW

File: routers/logoutrouter.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/**
+ * Created by piyush0 on 23/04/17.
+ */
+const router = require('express').Router();
+
+router.get('/', function (req, res){
+    req.session.destroy(function (err) {
```
Comment (championswimmer) [inline routers/logoutrouter.js]: also need to do req.logout()

Comment (championswimmer) [inline routers/logoutrouter.js]:
```js
    req.logout();
    req.session.destroy();
    res.redirect("/");
```

File: views/user/me.hbs [hunk 2]
```diff
@@ -5,3 +5,6 @@
     <a href="mailto:{{user.email}}">{{user.email}}</a>
 </div>
 
+<form method="get" action="/logout">
```
Comment (championswimmer) [inline views/user/me.hbs]: put this in the navbar on extreme right, if user is logged in

**Comment by piyush0** [comment]
Called all 3, req.user = null; req.logout(); req.session.destroy()

**Pull request merged**